build.gradle

```diff
@@ -454,10 +454,6 @@ dependencies {
     implementation "commons-fileupload:commons-fileupload:1.5"
     implementation "net.lingala.zip4j:zip4j:2.11.5"
 
-    implementation "org.jgrapht:jgrapht-core:1.5.2"
-    // use the latest version explicitly to avoid security vulnerabilities (currently Artemis and JPlag rely on jgrapht 1.5.2 which relies on apfloat)
-    implementation "org.apfloat:apfloat:1.14.0"
-
```

Comment:
We cannot really remove those dependencies. Due to transitive dependencies by JPlag, a version with security vulnerabilities would be used again, therefore, we need to explicitly mention those with the latest version here